code/game/gamemodes/blob/theblob.dm

```diff
@@ -20,6 +20,13 @@
 	var/atmosblock = 0 //if the blob blocks atmos and heat spread
 	var/mob/camera/blob/overmind
 
+/obj/structure/blob/attack_hand(mob/M)
+	. = ..()
+	var/a = pick("gently stroke", "nuzzle", "affectionatly pet", "cuddle")
+	M.visible_message("<span class='notice'>[M] [as] the \[src]!</span>", "<span class='notice'>You [b] the \[src]!</span>")
```

Comment:
Yeah, @MoreRobustThanYou, you were mistaken here. You also forgot to swap out the [b]. The "s" goes _outside_ of the [a], so that it places a and then appends the s to it.